### PR TITLE
Added snapmail server and database migration file

### PR DESCRIPTION
**Pull request by azharmalik3**
@ijunaid8088

### CODE REVIEW

File: lib/evercam_media/snapmail/poll_handler.ex [hunk 1]
```diff
@@ -0,0 +1,17 @@
+defmodule EvercamMedia.Snapmail.PollHandler do
+  @moduledoc """
+  TODO
```
Comment (marcoherbst) [inline lib/evercam_media/snapmail/poll_handler.ex]: TODO found

File: lib/evercam_media/snapmail/snapmailer_supervisor.ex [hunk 2]
```diff
@@ -0,0 +1,75 @@
+defmodule EvercamMedia.Snapmail.SnapmailerSupervisor do
+  @moduledoc """
+  TODO
```
Comment (marcoherbst) [inline lib/evercam_media/snapmail/snapmailer_supervisor.ex]: TODO found

**Comment by marcoherbst** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 3 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/evercam/evercam-server/pulls/576.

**Comment by ijunaid8088** [comment]
:+1: Everything seems fine.

**Pull request merged**